sstables_loader.cc

```diff
@@ -217,6 +217,8 @@ future<> sstable_streamer::stream_sstable_mutations(const dht::partition_range&
                     if (primary_replica_only && current_targets.size() > 1) {
                         current_targets.resize(1);
                     }
+                    auto pending = _erm->get_pending_endpoints(current_dk.token());
+                    std::move(pending.begin(), pending.end(), std::back_inserter(current_targets));
```

Comment:
This will break the case where the primary_replica_only is true but pending node is not a primary replica after the topology change.

Reply:
Perhaps it is better to serialize the load and stream with topology changes and tablet migrations of the table we are loading.

Reply:
@asias I don't understand why primary-replica-only is employed in load-and-stream. with repair -pr, we know repair will run on each node, so entire cluster will be repaired. but load-and-stream is push, not pull.

Reply:
The reason we have primary-replica-only flag is to reducing the amount of data we stream. Users could use primary-replica-only during load and stream , and run repair after it bring data to all replicas.